config/jobs/kubernetes/sig-storage/sig-storage-gce-config.yaml

```diff
@@ -307,3 +358,40 @@ periodics:
           memory: "6Gi"
   annotations:
     testgrid-num-columns-recent: '20'
+- interval: 24h
+  name: ci-kubernetes-e2e-snapshot
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+    preset-bazel-scratch-dir: "true"
+    preset-pull-kubernetes-e2e: "true"
+    preset-pull-kubernetes-e2e-gce: "true"
+  spec:
+    containers:
+    - args:
+      - --root=/go/src
+      - --repo=k8s.io/kubernetes=$(PULL_REFS)
+      - --repo=k8s.io/release
+      - --upload=gs://kubernetes-jenkins/pr-logs
+      - --timeout=150
+      - --scenario=kubernetes_e2e
+      - --
+      - --build=bazel
+      - --cluster=
+      - --extract=local
+      - --gcp-node-image=ubuntu
```

Comment:
Let's use --gcp-node-image=gci, and remove image-family and image-project args

Reply:
Done